tree/tree/src/TTree.cxx

```diff
@@ -8084,12 +8084,14 @@ void TTree::ResetBranchAddresses()
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Loop over tree entries and print entries passing selection.
+/// Loop interactively over tree entries and print entries passing selection.
 ///
 /// - If varexp is 0 (or "") then print only first 8 columns.
 /// - If varexp = "*" print all columns.
 ///
 /// Otherwise a columns selection can be made using "var1:var2:var3".
+///
+/// \see TTree::SetScanField for controlling how many rows are printed
 /// \see TTreePlayer::Scan for more information
 
 Long64_t TTree::Scan(const char* varexp, const char* selection, Option_t* option, Long64_t nentries, Long64_t firstentry)
```

Comment:
I believe the parameter `nentries` can be used to the same extent as `SetScanField`, but just for this particular call to `Scan`. Maybe we want to document that as well as it is not explicitly called out?

Reply:
Yes, but with the exception of nentries=0 and nentries=totalEntries, neither of which is the equivalent of SetScanField(0)